config/migrations/20240529120916-add-prov-activity-type-to-indieningsactiviteit.sparql

```diff
@@ -0,0 +1,17 @@
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX sign: <http://mu.semte.ch/vocabularies/ext/handtekenen/>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+
+INSERT { GRAPH ?g { ?s a prov:Activity } }
+WHERE {
+  VALUES (?type) {
+    (ext:Indieningsactiviteit)
+    (ext:AgendaStatusActivity)
```

Comment:
queried on ACC, `ext:AgendaStatusActivity` all have a `prov:Activity` already.
Doesn't really hurt to have it here anyway.